lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -265,7 +265,7 @@ sub start
         send_federation       => ( not $self->{workers} ),
         enable_media_repo     => ( not $self->{workers} ),
         run_background_tasks_on  => ( $self->{workers} ? "background_worker1" : "master" ),
-        worker_to_update_user_directory  => ( $self->{workers} ? "user_dir" : "null" ),
+        worker_to_update_user_directory  => ( $self->{workers} ? "user_dir" : undef ),
         # update_user_directory is kept for backwards compatibility,
         # worker_to_update_user_directory is prioritized before this option.
         update_user_directory => ( not $self->{workers} ),
```

Comment:
presumably once https://github.com/matrix-org/synapse/pull/11450, this will be removed?

Reply:
Not directly, i'd like to wait until most/all branches have been updated with the develop branch, else you'll get unrelated sytest errors :/

I'll make a reminder to revisit this in a few months or so, then enumerate all open PRs, and see if they include the commit in their branch or not. If it's not a lot of PRs left, ill just boop them with "hey, please update to `develop`", and throw a PR in here which removes this option.